pinot-common/src/main/java/com/linkedin/pinot/common/response/BrokerResponse.java

```diff
@@ -91,6 +91,11 @@ String toJsonString()
    */
   long getTotalDocs();
 
+  /**
+   * Returns whether the number of groups limit has been reached.
```

Comment:
can we link to the config that controls the max groups

Reply:
That is not possible because the config is in pinot-core... We hit this issue again lol